CHANGELOG.md

```diff
@@ -21,11 +21,11 @@ The following components are considered to be internal and they are excluded fro
 - `marker_adapter`
 - `marker_error`
 
-## [Unreleased]
+## [0.3.0] - 2023-10-05
```

Comment:
@Veetaha The release docs, say that this is the way to set the version for the next release. Is it fine to already include the link brackets `[]` and release date, or should the headline only be `## 0.3.0`?

Reply:
The link brackets *must* be present. The format of this header is parsed with brackets in mind. You did this right, the format has to follow the format of the version like below

Reply:
Perfect :heart: